third_party/move/move-bytecode-verifier/src/features.rs

```diff
@@ -55,7 +55,14 @@ impl<'a> FeatureVerifier<'a> {
         };
         verifier.verify_signatures()?;
         verifier.verify_function_handles()?;
-        verifier.verify_function_defs()
+        verifier.verify_function_defs()?;
```

Comment:
Why do we verify function defs here? Script has no functions, can this be removed? It is confusing to be checking it and then later having a RAC check for script

Reply:
Good catch. In fact it was also wrong, as the script.code wasn't verified.